app/views/themes/halcyonic/layouts/halcyonic.html.erb

```diff
@@ -35,9 +35,9 @@
 
       <%= render partial: 'themes/halcyonic/includes/footer' %>
 
-      <%# if user.is_admin? %>
-      <%= render partial: 'admin/includes/admin_toolbar' %>
-      <%# end %>
+      <% if current_user_can? 'view_admin_toolbar' %>
```

Comment:
Mmm I'm wondering if its good for consistency to put all authorisation behaviour through Pundit, including this for viewing the admin toolbar. 

```diff
-<% if current_user_can? 'view_admin_toolbar' %>
+<% if policy(:admin_toolbar).show? %>
```

I'm not sure really...If you did want to try, I think a headless policy would do the trick: https://github.com/varvet/pundit/blob/master/README.md#headless-policies